src/commands/Moderation/dehoist.ts

```diff
@@ -0,0 +1,49 @@
+import { CommandStore, KlasaMessage } from 'klasa';
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { PermissionLevels } from '../../lib/types/Enums';
+import { MessageEmbed } from 'discord.js';
+import { getColor } from '../../lib/util/util';
+import { api } from '../../lib/util/Models/Api';
+
+export default class extends SkyraCommand {
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			aliases: ['dh'],
+			cooldown: 5,
+			description: language => language.tget('COMMAND_DEHOIST_DESCRIPTION'),
+			extendedHelp: language => language.tget('COMMAND_DEHOIST_EXTENDED'),
+			runIn: ['text'],
+			permissionLevel: PermissionLevels.Moderator,
+			requiredPermissions: ['MANAGE_NICKNAMES']
+		});
+	}
+
+	public async run(message: KlasaMessage) {
+		let counter = 0;
+		const promises = [];
+		const members = message.guild!.memberTags;
+		const msg = await message.sendLocale('COMMAND_HIGHERLOWER_LOADING');
+
+		for (const [memberId, memberTag] of members) {
+			if (memberTag.nickname && memberTag.nickname.charCodeAt(0) < 65 /* That's A */) {
```

Comment:
That `65` should be a constant like:
```typescript
private readonly kLowestCode = 'A'.charCodeAt(0);
```

So we do this:
```suggestion
			if (memberTag.nickname && memberTag.nickname.charCodeAt(0) < this.kLowestCode) {
```